sonic-chassisd/scripts/chassisd

```diff
@@ -687,14 +791,42 @@ class ChassisdDaemon(daemon_base.DaemonBase):
 
         self.log_info("Shutting down...")
 
+
+class DpuChassisdDaemon(ChassisdDaemon):
```

Comment:
Only reason to inherit from ChassisDaemon to leverage signal handler correct?

Reply:
the reason is to inherit the interface of `DaemonBase`

Reply:
Why can't we inherit from daemon_base.DaemonBase directly?